src/js/components/bulkDownload/awards/AwardDataContent.jsx

```diff
@@ -149,6 +150,7 @@ export default class AwardDataContent extends React.Component {
                             currentLocation={awards.location}
                             updateFilter={this.props.updateFilter}
                             currentLocationType={awards.locationType} />
+                        <DefCodeFilter type="awards" />
```

Comment:
We need to update the user selections at the bottom of the form. Your selected options are...